cmd/tfimport/main.go

```diff
@@ -61,8 +62,17 @@ func run() error {
 		return fmt.Errorf("expand path %q: %v", *inputDir, err)
 	}
 
-	// Create Terraform command runners.
+	// Determine the runners to use.
 	rn := &runner.Default{}
+	var importRn runner.Runner
+	if *dryRun {
+		importRn = &runner.Dry{}
+		log.Printf("Dry run mode, logging commands but not executing any imports.")
+	} else {
+		importRn = &runner.Default{}
```

Comment:
just name is rn is sufficient

Reply:
I actually need to use a different runner for importing, since I still need to run `terraform init` and `terraform show` as normal. Should I rename it to `irn` maybe, to be shorter?

Reply:
Got it. Then importRn lgtm.